### ISSUE TITLE
NPE PlayerQuitEvent; ru_RU locale; debug paste doesnt give link

### ISSUE THREAD

**Issue by DardBrinza** [BedwarsRel/BedwarsRel#708]
Latest build spigot 1.8.8 and bedwars rel 1.3.4

PlayerQuitEvent
http://pastebin.com/3DLLqe7Q

ru_RU locale
http://joxi.ru/EA4p9KRfwDx9ZA
http://joxi.ru/52aQ8naI4GDak2?d=1

Sometimes my db spam this error
http://pastebin.com/at9kLXY9
http://pastebin.com/jx4kMwK5

I send 3-4 times commands debug and i get success message, but it doesnt give link
http://joxi.ru/GrqezYbcQNYEdr

**Comment by DardBrinza** [comment]
Up :(

**Comment by sebastianbinder** [comment]
We are quite busy at the moment in private life so please be patient as we manage this project completely in our free time.

**Comment by sebastianbinder** [comment]
> PlayerQuitEvent
http://pastebin.com/3DLLqe7Q
(If player banned/kicked from console, he remains in the game scoreboard/tab etc and game dont end)

I cannot imagine that this error happens frequently. If it happens, then maybe there is a bug in bukkit code, because even if a player gets banned to kicked from console the regular `PlayerQuitEvent` fires up. And then according to code the player leaving is being removed from the game.

> When player already kicked/banned by console, i try to kick him bw command
http://pastebin.com/VB9DPVCy

We included two checks: First if the player is online and second if the player is in a game. If one of those two checks fails, like in the scenario you described, we would not even reach that point where the error showed up.

> Sometimes my db spam this error
http://pastebin.com/at9kLXY9
http://pastebin.com/jx4kMwK5

This means that a duplicate key has been found in your database. Seems like there is something messed up with your setup, because I checked both UUIDs stated there and they are both no Minecraft UUIDs...

> I send 3-4 times commands debug and i get success message, but it doesnt give link
http://joxi.ru/GrqezYbcQNYEdr

The hastebin API seems to be broken. We know that we need to fix this.

> Cannot measure distance
http://pastebin.com/DmFpGs76

Somehow one player of that game is in another world so distance cannot be measured. Did you teleport him somewhere?

**Comment by DardBrinza** [comment]
> Somehow one player of that game is in another world so distance cannot be measured. Did you teleport him somewhere?
No, i use multiverse, game arena and waiting lobby in same world. Global waiting lobby: false.

> Server crash when game start. I have studied your code to remove entity on game end, it doesnt worked.
Not remove like tnt-sheep, resources etc
logs http://pastebin.com/mkbHWyUj
crash http://pastebin.com/bLD7EKSb
